### DISCUSSION TITLE
Create project from console

### DISCUSSION THREAD

**Discussion by zuczeq** [sorry-cypress/sorry-cypress#711] [Q&A]
Hi everyone!
I can't find a solution, maybe you can help.. i would like to create sorry-cypress projects from cli/console while running cypress tests. Is it possible? Creating them from webGui is not convinient.. Any one had the same problem?

**Comment by agoldis** [comment]
I think projects are created automatically if not exist already while getting results

**Reply by Lory1990** [reply]
I confirm
